handlers/anteHandler.go

```diff
@@ -185,3 +188,77 @@ func includeDepositAnteHandler(ctx sdk.Context, ds store.DataStore, msg msgs.Inc
 	}
 	return ctx, sdk.Result{}, false
 }
+
+// validates that the confirmSig msg is valid given the current plasma sidechain state
+func confirmSigAnteHandler(ctx sdk.Context, ds store.DataStore, confirmSigMsg msgs.ConfirmSigMsg, client plasmaConn) (newCtx sdk.Context, res sdk.Result, abort bool) {
+
+	// attempt to recover signers
+	signers := confirmSigMsg.GetSigners()
+	if len(signers) == 0 {
```

Comment:
I thought we were not expecting anyone to sign the actual confirmSigMsg struct? Anyone can broadcast the signatures

Reply:
We should instead be grabbing the confirmation hash of the transaction corresponding to the inputs and checking the confirm signature against that transaction hash

Reply:
Checking confirm sig happens in `validateConfirmSignatures()` call in `validateConfirmSigMsgInput()` now, removed need for signature on confirmSigMsg struct